docker/README.md

```diff
@@ -1,111 +0,0 @@
-# Docker for Gogs
```

Comment:
Because multiple parts don't match anymore and instead of this file I would like to add a chapter to the docs and the example deployment repo.